discovery-provider/plugins/pedalboard/apps/relay/src/validate.ts

```diff
@@ -0,0 +1,29 @@
+import { decodeAbi } from "./abi";
+import { logger } from "./logger";
+import { ManageEntityParameters } from "./types/entityManager";
+
+/// async in case we need to make a db call
+export const validateTransactionData = async (
+  encodedABI: string
+): Promise<ManageEntityParameters> => {
+  const decoded = decodeAbi(encodedABI);
+  // TODO: validate decoded tx with zod
```

Comment:
i don't think relay needs to do much validation just to keep things simple

Reply:
yeah not going to be included in this pr

Reply:
for my learning where would the validation happen if not here? in em?

Reply:
it would happen on indexer, this is hopefully reduce garbage requests and catch them at relay time instead of at indexing